### ISSUE TITLE
Voices audio channel

### ISSUE THREAD

**Issue by NoxArt** [antest1/GotoBrowser#8]
Hi, it seems like as soon as an audio track of a shipgirl's quote in a battle is to start playing, the currently played track is cancelled, so they are never taking at the same time. Not sure what information is relevant, feel free to ask. Android v9

**Comment by Oradimi** [comment]
I'm on Android 7.1.2, and there's the same "issue". But tbh that didn't really bother me.

**Comment by NoxArt** [comment]
Sometimes it gets quite prominent and noticeable

I'm not an Android dev, but briefly checking the source it seems (FullscreenActivity.java) GTB is handling playing audio itself and already handles this for sound effects. Maybe it's this way because of subtitles, but I think those could be still be reset so that the last one is showing, the previous audio tracks would keep playing until they're finished or get cycled out of the pool.

**Comment by NoxArt** [comment]
Weird, today it seemed like in PvP it didn't work, in PvE ... not sure, had a bad selection of voices (didn't overlay as much) and now I can't get to it atm. But it's unlikely they would behave differently I guess. Relaunch didn't help. Also voice volume in combat seemed kinda too low, not sure it's caused by my change, would seem likely. I'll look into both as soon as I can ... both worked well when I was debugging it though, sigh
(possibly a separate weird instance where music kept playing after switching apps until I moved through menu screens to get a different music ... but I don't think I touched code for music, I'll check this too though)
